src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/MissingAssemblyTests.cs

```diff
@@ -609,7 +612,11 @@ void M()
                 out errorMessage); 
  
             Assert.Equal(2, numRetries); // Ensure that we actually retried and that we bailed out on the second retry if the same identity was seen in the diagnostics.
-            Assert.Equal($"error CS0012: The type 'MissingType' is defined in an assembly that is not referenced. You must add a reference to assembly '{missingIdentity}'.", errorMessage); 
```

Comment:
This caused the unittest failure on  non-English machines, so I wanted to avoid string comparison but I will revert the change to leave it as it is. Instead, I will take "EnsureEnglishUICulture" approach. Thanks!